GDJS/GDJS/Events/CodeGeneration/BehaviorCodeGenerator.h

```diff
@@ -57,32 +57,69 @@ class BehaviorCodeGenerator {
     return "_set" + propertyName;
   }
 
+  /**
+   * \brief Generate the name of the method to get the value of the shared property
+   * of a behavior.
+   */
+  static gd::String GetBehaviorSharedPropertyGetterName(
+      const gd::String& propertyName) {
+    return "_getShared" + propertyName;
```

Comment:
We should probably ensure properties are not starting with "Shared"? To avoid conflicts in method names?

Reply:
I moved shared properties getter and setter in the shared data class to avoid name collisions.